README.md

```diff
@@ -25,11 +40,16 @@ user    0m1.003s
 sys     0m1.300s
 ```
 
-After dumping the SVN history, I make **svneverever** show a tree of
-all directories in there ever ever.
+The output can now be obtained by pointing **svneverver** to the svn directory.
```

Comment:
Can we say "svn store directory `bs2b_svn_store`" here? It think that would be a bit more clear.